### PR TITLE
Fixes ambient sounds playing in the lobby

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

Removes ambient sounds from the `/area/start` area, primarily so that they don't overlap with the lobby music.

## Why It's Good For The Game

It sounds confusing, and is most likely an unintentional change.

## Changelog
:cl:
fix: Fixed ambient sounds playing on the lobby/menu screen.
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
On one hand, it's a bug--on the other, it only occurs after round has started and it does have kinda an atmospheric vibe when you're just sitting on the screen.

Not sure about this or not, to be honest.

**Comment by SabreML** [comment]
> On one hand, it's a bug--on the other, it only occurs after round has started and it does have kinda an atmospheric vibe when you're just sitting on the screen.
> 
> Not sure about this or not, to be honest.

If there's no lobby music playing then sure, it might be atmospheric. But currently it just seems to clash horribly.
And you also have to consider anyone that joins after roundstart, since they get both.

**Review by dearmochi — APPROVED** [review]

**Comment by S34NW** [comment]
It just sounds weird and out of place, good riddance.

**Review by AffectedArc07 — APPROVED** [review]
Im in favour because ambience and lobby music clashing does sound bad, and this affects people who join the lobby mid round.

File: code/controllers/subsystem/ambience.dm [hunk 1]
```diff
@@ -21,8 +21,10 @@ SUBSYSTEM_DEF(ambience)
 
 		var/area/current_area = get_area(client_iterator.mob)
 
-		var/sound = pick(current_area.ambientsounds)
+		var/ambience = pick(current_area.ambientsounds)
+		if(!ambience)
```
Comment (AffectedArc07) [inline code/controllers/subsystem/ambience.dm:25]: This can result in `error: pick from empty list`

Consider using `safepick`

Comment (SabreML) [reply code/controllers/subsystem/ambience.dm:25]: Ah, I'd assumed that the `pick()` proc would just pass on `null` if it was empty.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**